app/src/views/private/components/filter-sidebar-detail/field-filter.vue

```diff
@@ -89,7 +90,7 @@ export default defineComponent({
 		return { activeOperator, value, name, parsedField };
 
 		function getFieldForKey(fieldKey: string) {
-			return fieldsStore.getField(props.collection, fieldKey);
+			return getFieldDeep(props.collection, fieldKey);
```

Comment:
`getField` itself already follows dot notation 🤔 How is this `getFieldDeep` util different?

https://github.com/directus/directus/blob/706dca1740cede473ae0ef6c23eaebed099336ba/app/src/stores/fields.ts#L236-L242

Reply:
The above only checks for one-collection. Lets patch field store getField to also recursively follow o2m